erpnext/accounts/report/general_ledger/general_ledger.py

```diff
@@ -350,8 +354,12 @@ def update_value_in_dict(data, key, gle):
 			if filters.get("group_by") != _('Group by Voucher (Consolidated)'):
 				gle_map[gle.get(group_by)].entries.append(gle)
 			elif filters.get("group_by") == _('Group by Voucher (Consolidated)'):
-				key = (gle.get("voucher_type"), gle.get("voucher_no"),
-					gle.get("account"), gle.get("cost_center"))
+				keylist = [gle.get("voucher_type"), gle.get("voucher_no"), gle.get("account")]
+				if filters.get("include_dims"):
+					for dim in get_accounting_dimensions():
+						keylist.append(gle.get(dim))
+				keylist.append(gle.get("cost_center"))
```

Comment:
Any specific reason for removing cost center from above and appending here?

Reply:
Only so that the order of the fields in the key will be the same as the order of the fields in the SQL query. Not sure that it makes any difference, but having a single standard order for using/querying the fields seemed reasonable as the least likely to cause issues if that list is used for anything else in future revisions of the code. If you'd like me to consolidate the cost center back into the first line where the key is set, just let me know and I will go ahead and do so.